### PR TITLE
Add release notes for 5.13.2

### PR DESCRIPTION
**Pull request by echlebek**
Signed-off-by: Eric Chlebek <eric@sensu.io>

### CODE REVIEW

File: content/sensu-go/5.13/release-notes.md [hunk 1]
```diff
@@ -39,11 +40,21 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
 
 ---
 
+## 5.13.2 release notes
+
+**September 19, 2019** &mdash; The latest release of Sensu Go, vison 5.13.2, is now available for download.
+This release is a stability release that fixes a bug for users that have the postgresql event store enabled.
```
Comment (cwjohnston) [inline content/sensu-go/5.13/release-notes.md]:
```suggestion
This release is a stability release that fixes a bug for users who have enabled the postgresql event store.
```

File: content/sensu-go/5.13/release-notes.md [hunk 2]
```diff
@@ -39,11 +40,21 @@ Read the [upgrade guide][1] for information on upgrading to the latest version o
...
+See the [upgrade guide][1] to upgrade Sensu to version 5.13.2.
+
+**Fixes:**
+
+- Metrics handlers now fire correctly when the postgresql event store is enabled.
```
Comment (cwjohnston) [inline content/sensu-go/5.13/release-notes.md]:
```suggestion
- Metrics handlers now correctly receive metric points when the postgresql event store is enabled.
```

File: content/sensu-go/5.13/release-notes.md [hunk 3]
```diff
@@ -7,7 +7,8 @@ version: "5.13"
 menu: "sensu-go-5.13"
 ---
 
-- [5.13.1 release notes](#5-13-0-release-notes)
```
Comment (hillaryfraley) [inline content/sensu-go/5.13/release-notes.md:10]: good catch

**Review by cwjohnston — CHANGES_REQUESTED** [review]

**Review by asachs01 — APPROVED** [review]
👍 LGTM

**Pull request merged**